go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/openfga/openfga
 
 go 1.18
 
+replace go.buf.build/openfga/go/openfga/api => /Users/jonwhit/github/openfga/api/proto
```

Comment:
Waiting on https://github.com/openfga/api/pull/21 to be merged before this can be wired up. For now this is pointing at my local module so that it builds against the new API definitions.